debug/nomap.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+var mapboxgl = require('mapbox-gl');
+var insertCss = require('insert-css');
+var fs = require('fs');
+
+mapboxgl.accessToken = window.localStorage.getItem('MapboxAccessToken');
+
+
+var meta = document.createElement('meta');
+meta.name = 'viewport';
+meta.content = 'initial-scale=1,maximum-scale=1,user-scalable=no';
+document.getElementsByTagName('head')[0].appendChild(meta);
+
+insertCss(fs.readFileSync('./lib/mapbox-gl-geocoder.css', 'utf8'));
+insertCss(
+  fs.readFileSync('./node_modules/mapbox-gl/dist/mapbox-gl.css', 'utf8')
+);
+
+var MapboxGeocoder = require('../');
+
+var notMapDiv = document.body.appendChild(document.createElement('div'));
+notMapDiv.style.position = 'absolute';
+notMapDiv.style.top = 0;
+notMapDiv.style.right = 0;
+notMapDiv.style.left = 0;
+notMapDiv.style.bottom = 0;
```

Comment:
To reproduce the double stuff add in:
```suggestion
notMapDiv.style.bottom = '50%';

var anotherNotMapDiv = document.body.appendChild(document.createElement('div'));
anotherNotMapDiv.style.position = 'absolute';
anotherNotMapDiv.style.top = '50%';
anotherNotMapDiv.style.right = 0;
anotherNotMapDiv.style.left = 0;
anotherNotMapDiv.style.bottom = 0;
anotherNotMapDiv.style.backgroundColor = 'green';
anotherNotMapDiv.classList.add('notAMap');
```

Reply:
I added a test to `test.ui.js`, so I'm not sure we need to add this to the debug page.